### PR TITLE
Update orders-api-overview.md

### PR DESCRIPTION
**Pull request by bc-tgomez**
Added to note field. Create an order shipment by using `id` in a `GET` request to for products with variants.

# [DEVDOCS-3380](https://jira.bigcommerce.com/browse/DEVDOCS-3380)

## Creating Orders Shipments
* Added to note field. Create an order shipment by using `id` in a get request to for products with variants.

### CODE REVIEW

**Review by bc-vincent-zhao — APPROVED** [review]
thank you 🙇

**Review by traci707 — APPROVED** [review]
LGTM

File: docs/api-docs/getting-started/api-status-codes.md [hunk 1]
```diff
@@ -30,15 +30,16 @@ The BigCommerce API responds to requests with different HTTP status codes depend
 
 | Code | Text | Purpose |
 |-|-|-|
-| **400** | **Bad Request** | Issued when a malformed request was sent.
+| **400** | **Bad Request** | Issued when a malformed request was sent. |
```
Comment (markcmurphy) [inline docs/api-docs/getting-started/api-status-codes.md:33]:
```suggestion
| **400** | **Bad Request** | Issued when a malformed request was sent. <br /> The request can not be completed due to a URL restriction. Check the URL for ports that may conflict with site permissions. |
```

Comment (bc-tgomez) [reply docs/api-docs/getting-started/api-status-codes.md:33]: Added suggested commit to stay in line with other instances of table structure in other articles.

**Comment by bc-tgomez** [comment]
Changes look good and make sense, thanks @slsriehl.

**Review by slsriehl — APPROVED** [review]
party like it's 2099

**Pull request merged**